### PR TITLE
squeeze the Print PMA

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
[Ticket TBD](linkToTicket)

Just a few small changes to save a bit more space on the print

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
just squeezing the print to save pages - saves 1 of 7 in portrait, but 4 of 12 in landscape

File: src/sass/_panelMeetingAgendas.scss [hunk 1]
```diff
@@ -241,6 +241,9 @@
   .tm-footer {
     display: none;
   }
+  .pma-print-close-icon {
+    display: none;
+  }
 }
```
Comment (elizabeth-jimenez) [inline src/sass/_panelMeetingAgendas.scss:247]: `@page {size: landscape}`

we could force landscape here (more pages but content looks better), but idk

File: src/sass/_panelMeetingAgendas.scss [hunk 2]
```diff
@@ -260,26 +263,19 @@
...
-    margin: 2px 0 2px 0px;
-    border-top: solid 5px $primary-gray-dark;
-    border-bottom: solid 5px $primary-gray-dark;
+    margin-left: -10px;
+    border-top: solid 3px $primary-gray-dark;
   }
   .pma-table-wrapper {
-    box-shadow: $tm-gray 6px 6px 20px;
-    margin-bottom: 15px;
-    break-inside: avoid;
```
Comment (M00staff) [inline src/sass/_panelMeetingAgendas.scss:282]: i think you wanna keep the break-inside

File: src/sass/_panelMeetingTracker.scss [hunk 3]
```diff
@@ -23,21 +23,11 @@
...
+  margin-left: 20px;
 }
 
 @each $status, $color in $panelStatuses {
   .panel-status--#{$status} {
     background-color: $color;
   }
 }
-
-@media screen and (max-width: 1200px) {
```
Comment (elizabeth-jimenez) [inline src/sass/_panelMeetingTracker.scss:35]: i think this was useful at some point, but it seems no longer 🤔

File: src/sass/_panelMeetingAgendas.scss (hunk 2 above)
Comment (M00staff) [reply src/sass/_panelMeetingAgendas.scss:282]: as avoid

Comment (elizabeth-jimenez) [reply src/sass/_panelMeetingAgendas.scss:282]: discussed in Slack - flipping back to `avoid` 👍

**Comment by elizabeth-jimenez** [comment]
unfortunately the toast doesn't go away after closing the print view; i can give it a try for a little tomorrow, but if not, no big deal, user can just hit x

File: src/sass/_panelMeetingAgendas.scss (hunk 1 above)
Comment (M00staff) [reply src/sass/_panelMeetingAgendas.scss:253]: my vote is for better looking over more pages - but i can't recall what the preference from the users was

Comment (elizabeth-jimenez) [reply src/sass/_panelMeetingAgendas.scss:253]: i guess this file isn't showing as outdated cause that snippet itself didn't change 🤔 
I added it, just heads up it isn't showing here

**Comment by elizabeth-jimenez** [comment]
I git messed up this PR, moving to this one: https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2814

**Pull request closed without merging**